core/src/com/dragons/game/utilities/BodyBuilder.java

```diff
@@ -0,0 +1,38 @@
+package com.dragons.game.utilities;
+
+import com.badlogic.gdx.physics.box2d.Body;
+import com.badlogic.gdx.physics.box2d.BodyDef;
+import com.badlogic.gdx.physics.box2d.FixtureDef;
+import com.badlogic.gdx.physics.box2d.PolygonShape;
+import com.badlogic.gdx.physics.box2d.World;
+
+import static com.dragons.game.utilities.Constants.PPM;
+
+public final class BodyBuilder {
```

Comment:
Dette er nice! Må bare snu om på tankegangen i forhold til de andre klassene! 

Kan være jeg tolker feil, men sånn det er implementert nå "leier" hver klasse denne til å lage bodies hos seg selv, i seg selv. Det som skjer da er at de må ta inn world som argument for å kunne passe det til denne. Det er analogt med å at hver husstand leier inn møbelsnekkeren hver gang bordet knekker fremfor å ta med bordet til snekkerens verksted og la han reparere det der. Problemet er rett og slett at snekkeren må ta med hele verkstedet sitt hjem til deg for å fikse bordet ditt!

Reply:
Nja uenig i at det bryter med noe som helst prinsipp, hvis man skal lage en body må man uansett ha World et sted. Viktigste her er loose coupling og denne implementasjonen har det

Reply:
Nei det er det prinsippet som brytes

Reply:
Nå har block en direkte referanse til world, og world får direkte referanse til block. Det er definisjonen på tight coupling!